test/runtests.jl

```diff
@@ -20,6 +20,16 @@ end
 @testset "LinQuadOptInterface" begin
     solver = LQOI.MockLinQuadOptimizer()
 
+    @testset "Printing" begin
+        io = IOBuffer()
+        show(io, solver)
+        seekstart(io)
+        output = String(take!(io))
+        @test output == "A LinQuadOptInterface model with backend:\n" *
```

Comment:
You can use `sprint(show, solver) == "..."` instead (see `io_test` in `JuMP/test/print.jl`)